strapi/strapi
# [Blocks Editor] Description and placeholder added

### What does it do?

Description and placeholder added for Blocks editor

### Why is it needed?

CTB configure the view option doesn't work

### How to test it?

In CTB , Set some placeholder and description value for the blocks type and check if its reflected in CM.
